integrations/operator/crdgen/schemagen.go

```diff
@@ -136,7 +136,10 @@ func (generator *SchemaGenerator) addResource(file *File, name string, opts ...r
 	} else {
 		specField, ok := rootMsg.GetField("Spec")
 		if !ok {
-			return trace.NotFound("message %q does not have Spec field", name)
+			specField, ok = rootMsg.GetField("spec")
+			if !ok {
+				return trace.NotFound("message %q does not have Spec field", name)
+			}
```

Comment:
Isn't it the same thing? :thinking:

Reply:
Spec with a capital S vs. spec with a lowercase S.  :-)

Reply:
🤯 
That is very subtle
Can we add a comment?